src/rpcclient.cpp

```diff
@@ -91,6 +91,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "estimatepriority", 0 },
     { "prioritisetransaction", 1 },
     { "prioritisetransaction", 2 },
+    { "setban", 2 },
```

Comment:
This is only for the rpc client (bitcoin-cli) string to json conversion which is not required for `listbanned`.